src/rte.cc

```diff
@@ -1887,11 +1887,13 @@ void rtmethods_jacobian_finalisation(
 
       else if (jacobian_quantities[iq].Mode() == "rh") {
         // Here x = (p_sat/p) * z
+        AtmField atm_field;
         Tensor3 t_data(ppvar_atm.nelem(), 1, 1);
         for (Index ip = 0; ip < np; ip++) {
           t_data(ip, 0, 0) = ppvar_atm[ip].temperature;
         }
-        water_p_eq_agendaExecute(ws, water_p_eq, t_data, water_p_eq_agenda);
+        ARTS_ASSERT(false)
+        water_p_eq_agendaExecute(ws, water_p_eq, atm_field, water_p_eq_agenda);
         for (Index ip = 0; ip < np; ip++) {
           diy_dpath[iq](ip, joker, joker) *= water_p_eq(ip, 0, 0) / ppvar_atm[ip].pressure;
         }
```

Comment:
@erikssonpatrick This is the part where we essentially break the idea behind `water_p_eq_agenda` that I mentioned at the meeting.  Had the `Agenda` only taken the atmospheric temperature or (better in by branch) the atmospheric point values, then this code would probably work better.  But the `Agenda` took `t_field`.  This is a field variable.  It is then using that field variable as if it is just a list of temperatures.  This is why the data has to be mangled so much here that we actually send in something that has nothing to do with it.

One example of breaking this is that you could easily imagine a function also taking the old VMR field and saying that the equivalent pressure is actually changed because we have a lot if IWC in the atmosphere.

I think the cleanest way to redesign this is that `water_p_eq_agenda` only takes a single `AtmPoint` in the future.  Of is the equivalent water pressure different in layer 0 as a function of layer 1?

Reply:
The saturation pressure of water only depends on the local temperature, and the cleanest solution would be that the agenda takes a single temperature as input. We then need a WSV for it. Should we introduce a_temperature?